README.md

```diff
@@ -16,10 +16,10 @@ More info about the core library can be found at the official gramine [docs](htt
 #### RATLSCreateKeyAndCrtDer
 
 ```go
-func RATLSCreateKeyAndCrtDer() error
+func RATLSCreateKeyAndCrtDer(crtPath, keyPath string) error
 ```
 
-Creates RA-TLS Certificate and Key required for remote attestation at location specified using enviornment variables: `RATLS_CRT_PATH` and `RATLS_KEY_PATH`.
+Creates RA-TLS Certificate and Key required for remote attestation at location specified using following arguments: `RATLS_CRT_PATH` and `RATLS_KEY_PATH`.
```

Comment:
Arguments are still in env var format.

Reply:
![image](https://user-images.githubusercontent.com/35560828/225848674-f7487352-7b11-467c-934a-f46c3b52aac7.png)